site/getting-started/setup-app.md

```diff
@@ -1,153 +0,0 @@
----
-outline: deep
-head:
-  - - meta
-    - property: og:title
-      content: Getting started guide
-  - - meta
-    - name: description
-      content: Learn how to get started with Alchemy's Embedded Accounts using Account Kit and the Alchemy Signer, Modular Account, Rundler and Gas Manager.
-  - - meta
-    - property: og:description
-      content: Learn how to get started with Alchemy's Embedded Accounts using Account Kit and the Alchemy Signer, Modular Account, Rundler and Gas Manager.
-  - - meta
-    - name: twitter:title
-      content: Getting started guide
-  - - meta
-    - name: twitter:description
-      content: Learn how to get started with Alchemy's Embedded Accounts using Account Kit and the Alchemy Signer, Modular Account, Rundler and Gas Manager.
----
-
-# Setup your application
-
-This section will walk you through the steps to create your application, install the necessary dependencies, and write the boilerplate code you'll need to integrate Embedded Accounts!
-
-## Create your app
-
-First, create a [NextJS React application](https://nextjs.org/docs/pages/api-reference/create-next-app) by running `npx create-next-app` from your terminal in a project directory of your choice. Select the default configuration for each question the `create-next-app` CLI.
-
-Also, at the root of this new application directory, add a `.env` file.
-
-## Configure the Embedded Accounts
```

Comment:
I think we still want this section in the new docs somewhere so users know how to get setup with the keys and configs they need

Reply:
Good idea. It might even be better for the READMEs of various starter apps to refer here so we have it all in one place, rather than replicating it everywhere.